### PR TITLE
Fixed too long text

### PR DESCRIPTION
**Pull request by Himura2la**
## Description
I changed "Username or email" to "Login or Email"

## Motivation and Context
The text "Имя пользователя или Email" does not fit its field. It should be smaller. 

## Screenshots (if appropriate):
http://screencast.com/t/gUkxW86ljm3

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: core/l10n/ru.json [hunk 1]
```diff
@@ -321,4 +321,4 @@
     "This %s instance is currently in maintenance mode, which may take a while." : "Этот сервер %s находится в режиме технического обслуживания, которое может занять некоторое время.",
     "This page will refresh itself when the %s instance is available again." : "Эта страница автоматически обновится, когда сервер %s снова станет доступен."
 },"pluralForm" :"nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n%100>=11 && n%100<=14)? 2 : 3);"
-}
```
Comment (Himura2la) [inline core/l10n/ru.json]: I used the online github editor, this was unintentional

**Comment by DeepDiver1975** [comment]
Translations have to be fixed on transifex - THX

**Comment by DeepDiver1975** [comment]
https://www.transifex.com/owncloud-org/owncloud/language/ru/

**Comment by Himura2la** [comment]
Thanks for the link

**Pull request closed without merging**